src/components/tabs/tabs.module.css

```diff
@@ -0,0 +1,55 @@
+/*
+NOTE: This file mainly uses selectors to apply styles because it ensures that
```

Comment:
I really like this approach 💖 we don't do it a ton in other components, I for one will definitely try to adopt this approach elsewhere 💯